src/tools/run-make-support/src/lib.rs

```diff
@@ -7,6 +7,8 @@
 // skip rustfmt on this file.
 #![cfg_attr(rustfmt, rustfmt::skip)]
 
+pub use serde_json;
+
```

Comment:
Remark: is github diff outdated here or is this duplicated?

Reply:
It was actually simpler: I messed up my rebase. Fixed.